### DISCUSSION TITLE
SKXamlCanvas is blue on iOS simulator

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#9462] [Q&A]
[_From Julius/Techo Caninei#1903 on Discord_:](https://discord.com/channels/372137812037730304/550416151172087808/1006819478878822461)

Hey all, I am using a SKCanvas to draw an inage and some shapes to the screen. 

On android it works 100% but on the iPhone simulator (I will have an iphone to test on soon) it just shows blue (the background color from xaml).

Any idea how I could approach this? 

https://gist.github.com/juliusdeblaaij/31f33104e3c80edbda5e32e9a6250c89

**Comment by jeromelaban** [comment]
Have you tried running [this sample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/SkiaSharpTest) to see if it exhibits the same behavior?

**Reply by juliusdeblaaij** [reply]
The issue is fixed. 
1. In XAML my control had a Background tag set to Blue. 
2. As a result, I could not see any images or shapes on my iOS simulator.
3. When I changed it to transparant, as is the case in your sample code, I can see my markers and image. 😄 

For the sake of future users, I would like to know **why** this is the case, the docs did not mention it (assuming I checked the right pages).
If anyone is able to answer why, that would be very handy.
It can then possible be documented, if it is not already.
[https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.control.background?view=windowsdesktop-6.0](Microsoft Docs)
[https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.visualelement.background?view=xamarin-forms](Xamarin Docs)

**Reply by jeromelaban** [reply]
This is a behavior exposed by `SKXamlCanvas` in SkiaSharp, and in general, the background of the surface is handled through Skia primitives, which is why we did not see this before. Note that the two links you provided are not related to Uno (WPF and Xamarin.Forms in this case), and [this is one](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.panel.background?view=winrt-22621) is the correct property.
